tests/php81-compatibility.patch

```diff
@@ -374,3 +422,50 @@ index 6e5cca4..1d24bb6 100644
      {
      }
  
+@@ -23,7 +23,7 @@ class TrailingCommaOnFunctions
+     {
+         $var = null;
+ 
+-        $singleLine = static function (int $arg) use ($var): void {
++        $singleLine = static function (int $arg) use ($var,): void {
```

Comment:
This really doesn't look good :thinking:

Reply:
This is patched input so I can test `,` removal.

```diff
- $singleLine = static function (int $arg) use ($var,): void { // input
+ $singleLine = static function (int $arg) use ($var): void { // result
```